voronoi/voronoi/voronoi.R

```diff
@@ -0,0 +1,44 @@
+library(plyr)
```

Comment:
Top level comment would be helpful, but it's small and clearly named [NO CHANGE]

Reply:
Added